src/constants.js

```diff
@@ -47,3 +47,6 @@ export const LOGO_CLASS = {
   CARD: ("paypal-logo-card": "paypal-logo-card"),
   LOGO_COLOR: ("paypal-logo-color": "paypal-logo-color"),
 };
+
+const PACKAGE_VERSION = "";
```

Comment:
Since we have apps like checkout-components reading directly from source, should we always have a version included here? Assuming it gets replaced by the release process in both the `src` and the `dist`
```suggestion
const PACKAGE_VERSION = "2.1.1";
```